lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -129,47 +131,116 @@ class TestJumpTo(ModuleStoreTestCase):
     """
     Check the jumpto link for a course.
     """
-    MODULESTORE = TEST_DATA_MIXED_MODULESTORE
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
```

Comment:
Right! I wanted the test cases to all default to Split, only testing Old Mongo in a couple places to make sure Legacy courseware still works with it.

(This has a nice side-effect of making our job easier when we tackle Old Mongo removal, since it's one less test case we have to flip over to using Split by default).

Reply:
Ah - right! I remember now that it specifies the default and can be overridden - thanks.